modules/service/src/main/scala/lucuma/itc/service/ItcCacheOrRemote.scala

```diff
@@ -94,17 +95,28 @@ trait ItcCacheOrRemote extends Version:
   ): F[TargetGraphsCalcResult] =
     cacheOrRemote(request, requestGraphs(itc))("itc:graph:spec", redis)
 
-  private def requestSpecTimeCalc[F[_]: Functor](itc: Itc[F])(
+  private def timeAndCountModeNotImplemented[F[_]: MonadThrow, A]: F[A] =
+    MonadThrow[F].raiseError[A](
```

Comment:
Would it make sense to mock this with a result of the requested time and count plus a made up s/n, let's say 1?

Reply:
As long as we don't get distracted and leave it that way I don't see why not.  Would that help get everything up to date?

Reply:
Well, it would let you continue on the backend as well as the UI, right?
I don't expect the implementation to take more than a few days

Reply:
I think it's fine to leave it throwing an unimplemented exception until it is ready.  The backend tests all use fake ITC results anyway so it doesn't stop us.